nodes/src/nodes/audio_tts/tts_engine.py

```
@@ -0,0 +1,508 @@
+            os.close(fd)
+        with wave.open(wav_path, 'wb') as wavf:
+            wavf.setnchannels(1)
+            wavf.setsampwidth(2)
+            wavf.setframerate(int(sampling_rate))
+            wavf.writeframes((audio_arr * 32767).astype(np.int16).tobytes())
+        if output_format == 'mp3':
+            self._transcode_wav_to_mp3(wav_path, out_path)
+            try:
+                os.remove(wav_path)
+            except OSError:
+                pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)